requests/__about__.py

```diff
@@ -0,0 +1,7 @@
+title = 'requests'
+version = '2.14.2'
+build = 0x021402
+author = 'Kenneth Reitz'
+license = 'Apache 2.0'
+copyright = 'Copyright 2017 Kenneth Reitz'
+cake = u'✨ 🍰 ✨ Thanks for using my software. It means the world to me.  --kennethreitz'
```

Comment:
It looks like the file may need `# -*- coding: utf-8 -*-` at the top for Python 2 because of this line.